openmc/filter.py

```diff
@@ -516,7 +516,7 @@ def get_bin(self, bin_index):
 
         return filter_bin
 
-    def get_pandas_dataframe(self, data_size, summary=None):
+    def get_pandas_dataframe(self, data_size, distribcell_paths=True):
```

Comment:
The only problem with making this `True` by default is that if the `Summary` object is not linked it will fail.